content/posts/May-2020.md

```diff
@@ -4,6 +4,40 @@ date: 2020-05-06
 draft: false
 ---
 
+## Week 22 (May 25th - May 29th)
+
+### packit
+
+* [The problem with kerberos initialization](https://github.com/packit-service/packit/issues/837)
+  was fixed. ([#838])
+* We released a new version of the packit package. ([release 0.11.1])
+* We have been also working on some smaller issues and preparation steps for our future work.
```

Comment:
this is pretty vague, please at least hint about the future work and why we are doing it

Reply:
Do we really want to have it in the blog? The ones that make sense to me was already commented.

I just want to:
- provide links
- say that the PRs above was not the only thing we are working on
- not go into the details

Reply:
oki, LGTM then